### PR TITLE
Admin All Users Page - just need database connection

### PR DESCRIPTION
**Pull request by jennacioffi**
Becca and I created a button on the Login page to access the all users admin page, this is a temporary button until the admin page is completed and permissions are created to get into an admin account.

There is temporary users currently held in the all-users page until we understand how to connect it to the database.

We also edited index.js under the router directory. There was a merge conflict, but it was just two routes added by us and another person, so I accepted both and they are both in place now as they didn't really conflict.

### CODE REVIEW

**Comment by jennacioffi** [comment]
I screwed up I'm trying to fix my own repo do not pull this rn lol my bad

**Comment by lynnbridge** [comment]
Will you let me know when this is ready to review?

**Comment by rebeccamacias** [comment]
@lynnbridge We need the backend PR to be merged before we can get this PR ready for review.

**Comment by rebeccamacias** [comment]
This is ready for review

File: frontend/src/views/AdminAllUsers.vue [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                    <td>{{ user.Role }}</td>
+                </tr>
+            </tbody>
+        </v-table>
+    </div>
+</template>
+
+
+<script>
+//TODO: errors pop up when trying to import this, I think it's a backend issue?
```
Comment (jebidiah252) [inline frontend/src/views/AdminAllUsers.vue:22]: What error is popping up here? I'm trying this out and getting errors about axios not being able to be imported.

Comment (rebeccamacias) [reply frontend/src/views/AdminAllUsers.vue:22]: Yes, those were the same ones I was seeing. I commented it out for now and put in the dummy data. It looks like your PR has the fixes to axios. 👍

**Review by lynnbridge — APPROVED** [review]

**Pull request merged**